Exercises/arrays/arrays.js

```diff
@@ -0,0 +1,35 @@
+//Make a function which will generate array with random length and random values.
+
+// eslint-disable-next-line array-callback-return
+function randomArray() {
+  return Array.from(new Array(Math.floor(Math.random() * 100)), () => { return Math.floor(Math.random() * 10); });
+}
+
+//Make a function which will take array and return copy of it.
+
+function arrayCopy(array) {
+  return array.slice();
+}
+
+//Make a function which will take array and element and return a index of element in array.
+
+function indexOfElement(array, element) {
+  return array.includes(element) ? `element has index #${array.indexOf(element)}` : 'no such element in array';
+}
+
+//Make function which will take two arrays and return association of them.
+
+function arrayAssociation(array1, array2) {
+  return array1.concat(array2);
+}
+
+//Make function that will change element in array by index using reference, and other one - pure function.
+
+function arrayChange(array, index) {
+  array.splice(index, 1, Math.floor(Math.random() * 100));
+  return array;
+}
+
+function arrayCopy2(array) {
+  return array.slice();
+}
```

Comment:
It should be not any pure function but pure function which returns new array with modified element.